test/sharness/t0010-basic-commands.sh

```diff
@@ -102,6 +102,19 @@ test_expect_failure "All ipfs commands docs are 80 columns or less" '
   fi
 '
 
+test_expect_success "All ipfs commands fail when passed a bad flag" '
+  echo 0 > fail
+  while read -r cmd
+  do
+    test_must_fail $cmd --badflag </dev/null >/dev/null ||
```

Comment:
As with `--help` (https://github.com/ipfs/go-ipfs/issues/4841#issuecomment-374742799), we shouldn't close stdin here.

Reply:
OkI will change that.

Reply:
Done.